cmd/wellknownoperator_test.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018-2019 The NATS Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ListWellKnownOperators(t *testing.T) {
+	wko, err := GetWellKnownOperators()
+	require.NoError(t, err)
+	require.NotNil(t, wko)
+	require.True(t, len(wko) >= 1)
+
+	s, err := FindKnownOperator("SYNADIA")
```

Comment:
Maybe add checks for the URLs, even though it is a bit a==a it might be worth the safety in case we change to a text file or something.

Reply:
not following

Reply:
I was just thinking there is value in having a double check that we set the URLs correctly in the code that specifies the known operators, so if that could changes we know we didn't break something